stellar_rust_sdk/src/offers/offers_for_account_request.rs

```diff
@@ -9,9 +11,19 @@ pub struct OfferAccountId(String);
 pub struct NoOfferAccountId;
 #[derive(Default)]
 
+#[derive(Pagination)]
 pub struct OffersForAccountRequest<I> {
     /// The ID of the account for which the offers are to be retrieved.
     account_id: I,
+    /// A pointer to a specific location in a collection of responses, derived from the
+    /// `paging_token` value of a record. Used for pagination control in the API response.
+    cursor: Option<u32>,
+    /// Specifies the maximum number of records to be returned in a single response.
+    /// The range for this parameter is from 1 to 200. The default value is set to 10.
+    limit: Option<u8>,
+    /// Determines the [`Order`] of the records in the response. Valid options are [`Order::Asc`] (ascending)
+    /// and [`Order::Desc`] (descending). If not specified, it defaults to ascending.
+    order: Option<Order>,
```

Comment:
Top!
Wat me wel opvalt als ik naar [dit endpoint](https://developers.stellar.org/docs/data/horizon/api-reference/get-all-offers) kijk is dat er nog meer properties meegegeven kunnen worden, zoals:

* selling_asset_type
* selling_asset_issuer
* selling_asset_code
* buying_asset_type
* buying_asset_issuer
* buying_asset_code

Moeten deze er nog bij (in een separate PR) of heb ik het nu over een ander endpoint?

Reply:
De documentatie is iets anders ingedeeld dan de endpoint groups in bv Laboratory. In dit geval staat 'offers for account' onder 'account': https://developers.stellar.org/docs/data/horizon/api-reference/get-offers-by-account-id

Is dit wat je bedoelt?

Reply:
Hmm, dat denk ik wel, want  [https://github.com/Baseflow/stellar-rust-sdk/blob/5ad5148474bdaca3ebada94c08581e49283e0d30/stellar_rust_sdk/src/offers/all_offers_request.rs#L38](AllOffersRequest) biedt wel ongeveer dezelfde waarden als de [documentatie](https://developers.stellar.org/docs/data/horizon/api-reference/get-all-offers) van stellar (maar niet helemaal 1-op-1 lijkt het). Volgens mij is dat het dan inderdaad.

Ik zou alleen dan denken dat deze onder de account module zou staan, gezien het endpoint in de documentatie, maar het lab doet weer iets anders. 
Voor de gebruiker maakt het onder aan de streep niets uit. Laat het voor nu maar even zo staan dan.